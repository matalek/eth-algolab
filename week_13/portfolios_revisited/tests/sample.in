1 1
2000 160
10000
1000 2500
3 4
2000 160
2000 160
2000 60
10000 0 0
0 10000 0
0 0 400
1000 2500
1000 1250
1000 466
1000 64
0 0
