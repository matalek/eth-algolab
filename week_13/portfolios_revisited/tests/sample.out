80
80
80
60
30
